tests/blackbox/data/validation/kyma_app_serverless_syntax_err.yml

```diff
@@ -0,0 +1,55 @@
+description: This mock response contains a complete deployable yaml-formatted manifest for a serverless function
+scenario_id: kyma_app_serverless_syntax_err
+mock_response_content: |-
+  "There seems to be a syntax error in the serverless function 'func1'. The function calls Dates() instead of Date(). Here is a corrected version of the function:
```

Comment:
we may not need the quotation mark `"` for the text.

Reply:
Where exactly? There is a closing quotation mark in line 40.